ow_plexil/src/plans/TestPanTilt.plp

```diff
@@ -0,0 +1,66 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Simple plan to test Pan and Tilt functions
+
+#include "plan-interface.h"
+
+TestPanTilt: Sequence
+{
+
+    log_info("Testing Pan, should succeed on all tests.");
+    LibraryCall Pan(Degrees = 65);
+    log_info("Current Pan Degree: ", Lookup(PanDegrees));
+    log_info("Current Tilt Degree: ", Lookup(TiltDegrees));
```

Comment:
Since the Plexil node prints the target pan/tilt, this is good output for visual comparison, however this is a bit tedious.  With a few more lines you could make this plan self-evaluating by comparing the target with actual (within the same tolerance used in the node).   A single test could be factored into a Library node, thus simplifying this plan as well.